Northstar.CustomServers/mod/scripts/vscripts/_loadouts_mp.gnut

```diff
@@ -78,6 +93,24 @@ void function DelayDestroyDroppedWeapon( entity weapon )
 		weapon.Destroy()
 }
 
+void function TryAlwaysDestroyDroppedWeapon()
+{
+	thread TryAlwaysDestroyDroppedWeapon_Threaded()
+}
+
+void function TryAlwaysDestroyDroppedWeapon_Threaded()
+{
+	svGlobal.levelEnt.Signal( "WeaponDestroyThink" )
+	svGlobal.levelEnt.EndSignal( "WeaponDestroyThink" ) // don't loop over many times
+
+	while( true )
```

Comment:
this is really not great, absolutely a last resort solution

Reply:
yeah but how do we modify anim events